.github/workflows/cargo_flake.yml

```diff
@@ -17,8 +18,11 @@ env:
 
 jobs:
   hack:
-    runs-on: [self-hosted, linux, x64, general]
     name: Cargo Flake
+    runs-on: [self-hosted, linux, x64, general]
+    strategy:
+      matrix:
+        target: [bench, bench-remap-functions, bench-remap, bench-wasm, bench-languages, bench-metrics]
```

Comment:
I think you dropped `default`. Or did you mean to do that?

Reply:
D'oh

Reply:
Corrected in https://github.com/timberio/vector/pull/7776/commits/322d1a2e318d1098b548936b0777bfcaed759dbf